lib/config.ts

```diff
@@ -309,6 +309,15 @@ export interface Config {
      */
     maxSessions?: number;
 
+    /**
+     * Whether or not to buffer output when running tests on multiple browsers
+     * in parallel. By default, when running multiple browser sessions results
+     * are buffered and not logged until the test run finishes. If true, when
+     * running multiple sessions in parallel results will be logged when each
+     * test finishes.
+     */
+    verboseMultiSessions: boolean;
```

Comment:
This is an optional interface property and should be listed as: `verboseMultiSessions?: boolean;`